### ISSUE TITLE
Ollama URL

### ISSUE THREAD

**Issue by jguhlin** [jeremychone/rust-devai#15]
### Issue Description

Thanks for adding in Ollama support and an example. How would I set the URL to a remote Ollama instance?

**Comment by jeremychone** [comment]
Yes, this is coming at some point. First, I need to update GenAI with a custom endpoint, and then allow it to be configured via the config.toml.

It will take a few weeks at least to get that done with the rest of the backlog. But this is a priority.

**Comment by jeremychone** [comment]
@jguhlin So, I added the `ServiceTargetResolver` in [genai](https://crates.io/crates/genai) v0.1.5, now, I need to find a good way to express that in devai.
